shell/src/main/java/alluxio/cli/docgen/ConfigurationDocGenerator.java

```diff
@@ -119,9 +155,19 @@ public static void writeCSVFile(Collection<? extends PropertyKey> defaultKeys, S
     } finally {
       try {
         closer.close();
+        if (validate) {
+          compareFiles(CSV_FILE_NAMES, filePath, "CSV");
```

Comment:
similar to how MetricsDocGenerator defines constants for CSV and YML suffixes, replace "CSV" with a constant. in general, be wary about using strings within code, unless they have little significance (ex. whitespace, newlines)